### DISCUSSION TITLE
What variable is available for use ?

### DISCUSSION THREAD

**Discussion by benoitrolland** [Mogztter/asciidoctor-web-pdf#614] [General]
When using asciidoctor-pdf, one can insert {section-title} or {chapter-title} from within the css-like theme.yml file.
Is it possible to use any similar variable in the css, when using asciidoctor-web.pdf ?

**Comment by ggrossetie** [comment]
The closest equivalent is probably `string-set` and `content`: https://www.w3.org/TR/css-gcpm-3/#content-function-header

**Reply by benoitrolland** [reply]
Thank you very much, 
I could insert chapter name on left pages and sub-chapter on right pages like with asciidoctor-pdf : 
```
h4>.anchor~.link { string-set: current-sub-chapter content() }
@top-center {
    content: string(current-sub-chapter, first);
}
```
But I am scratching my head to find a simple solution to preserve the (html elements and) style like done by default with asciidoctor-pdf.

Would it be possible using css to apply that string-set to a [running element](https://github.com/Mogztter/asciidoctor-web-pdf#running-elements)?

=> ok to a [docinfo ](https://github.com/Mogztter/asciidoctor-web-pdf#docinfo) element should do the trick ;-)
